src/main/java/com/cloudbees/jenkins/GitHubTrigger.java

```diff
@@ -15,6 +15,7 @@
  * and triggers a build.
  *
  * @author aaronwalker
+ * @deprecated extend {@link GitHubTrigger2} instead
```

Comment:
Doesn't show as deprecated in my IDE
<img width="672" alt="screen shot 2017-02-07 at 19 10 08" src="https://cloud.githubusercontent.com/assets/209336/22706974/168c58ac-ed69-11e6-8663-c3bc47e42925.png">